orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteExecutionHandler.kt

```diff
@@ -85,17 +85,13 @@ class CompleteExecutionHandler(
         }
       }
       log.debug("Execution ${execution.id} is with ${execution.status} status and  Disabled concurrent executions is ${execution.isLimitConcurrent}")
-        if (execution.isLimitConcurrent) {
-          if (execution.status != RUNNING) {
-            execution.pipelineConfigId?.let {
-              queue.push(StartWaitingExecutions(it, purgeQueue = !execution.isKeepWaitingPipelines))
-            }
-          } else {
-            log.debug("Not starting waiting executions as execution ${execution.id} is currently RUNNING with Disabled concurrent executions.")
-          }
-        } else {
-            log.debug("Execution ${execution.id} is not Disabled for concurrent executions, no need to run waiting executions")
+      if (execution.status != RUNNING) {
+        execution.pipelineConfigId?.let {
+          queue.push(StartWaitingExecutions(it, purgeQueue = !execution.isKeepWaitingPipelines))
         }
+      } else {
+        log.debug("Not starting waiting executions as execution ${execution.id} is currently RUNNING.")
+      }
       }
```

Comment:
Could you fix the indentation here please?

Reply:
Done